Sources/BSON/BSON.swift

```diff
@@ -211,6 +211,67 @@ extension BSON {
     }
 }
 
+extension BSON {
```

Comment:
ditto this doesn't seem related to the ticket at hand here, are these needed for the `ObjectID` tests or something?

Reply:
Yea to both comments, this was the last ticket that was carrying around 50+ commits of messy history I learned my rebase lesson on this project
Anyway cleaned it up, this just has OID changes now